src/module.c

```diff
@@ -142,6 +142,11 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
 		return REDISMODULE_ERR;
 	}
 
+	if(RedisModule_CreateCommand(ctx, "graph.RO_QUERY", CommandDispatch, "readonly", 1, 1,
```

Comment:
I think the flags should be `readonly deny-oom`.

Reply:
From [redis module api](https://redis.io/topics/modules-api-ref)
Any idea if the following might be relevant ?
```
"allow-stale": The command is allowed to run on slaves that don't serve stale data. Don't use if you don't know what this means.
```